benchmarks/src/kotlinMain/kotlin/io/rsocket/kotlin/benchmarks/RSocketKotlinBenchmark.kt

```diff
@@ -27,31 +27,33 @@ import kotlin.random.*
 
 class RSocketKotlinBenchmark : RSocketBenchmark<Payload>() {
 
-    lateinit var client: RSocket
+    lateinit var client: RSocketRequester
     lateinit var server: Job
 
     lateinit var payload: Payload
     lateinit var payloadsFlow: Flow<Payload>
 
+    fun payloadCopy(): Payload = payload.copy()
```

Comment:
This sort of pattern, reuse of a payload probably deserves some nicer more natural mechanism.  This is likely a source of bugs under error conditions which will be missed on the happy path.  How do we improve this? e.g. payload.permanent()?

Reply:
and how `payload.permanent()` will work? have just possibility to reuse data? and release after all?
@altavir has much experience working with kotlin with IO, and now researching adding rsocket-kotlin to his own projects. Maybe hi have some ideas on better payload handling?

Reply:
Just from my side. We do allocate a ByteBuf once, and then retain it every time we use it in benchmarks. Sounds like a copy but not really, thus not sure Kotlin IO can do the same